docs/en/Example_Batch.md

```diff
@@ -0,0 +1,76 @@
+# Batch Message Sample
+------
+Sending messages in batch improves performance of delivering small messages. Messages of the same batch should have: same topic, same waitStoreMsgOK and no schedule support. Besides, the total size of the messages in one batch should be no more than 1MiB.
```

Comment:
Besides, the total size of the messages in one batch should be no more than 4MiB. If a large number of messages are sent continuously, it is recommended that each batch message be less than 1MiB .